scripts/github_actions_real_implementation.py

```
@@ -0,0 +1,298 @@
+        src_files = 0
+        test_files = 0
+        total_lines = 0
+        
+        for root, dirs, files in os.walk('src'):
+            for file in files:
+                if file.endswith('.py'):
+                    src_files += 1
+                    try:
+                        with open(os.path.join(root, file), 'r') as f:
+                            total_lines += len(f.readlines())
+                    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)